src/components/reusable/modal/modal.scss

```diff
@@ -169,4 +172,5 @@ h1 {
   overflow-y: auto;
   margin: -4px;
   padding: 4px 16px 4px 4px;
+  background: var(--kd-color-background-accent-subtle);
```

Comment:
I don't think we actually want a background color on the body here. I think that was just done in Figma to indicate a slot.

Reply:
Sure @brian-patrick-3 . Will check with Chandan

Reply:
Confirmed with UX. Removed the background color on the body